### ISSUE TITLE
py27-lint-readme failing with Duplicate explicit target name:

### ISSUE THREAD

**Issue by peterjc** [galaxyproject/planemo#524]
Part of #523, the TravisCI TOX target py27-lint-readme is failing, e.g.

```
$ tox -e py27-lint-readme
```

or more verbose output with:

```
$ make lint-readme
if [ -f .venv/bin/activate ]; then . .venv/bin/activate; fi; python setup.py check -r -s
running check
warning: check: Duplicate implicit target name: "conda".

warning: check: Duplicate implicit target name: "docker".

warning: check: Duplicate explicit target name: "02a08a0".

warning: check: Duplicate explicit target name: "f3c6917".

warning: check: Duplicate explicit target name: "9427b47".

warning: check: Duplicate explicit target name: "pull request 216".

warning: check: Duplicate explicit target name: "pull request 207".

warning: check: Duplicate explicit target name: "8117e03".

warning: check: Duplicate explicit target name: "pull request 99".

warning: check: Duplicate explicit target name: "cwl".

error: Please correct your package.
make: *** [lint-readme] Error 1
```

Some look like trivially duplicated links can be confirmed with grep, e.g.

```
$ grep -i "02a08a0" *.rst
HISTORY.rst:* Add ``--extra_tools`` option to ``serve`` command. 02a08a0_
HISTORY.rst:.. _02a08a0: https://github.com/galaxyproject/planemo/commit/02a08a0
HISTORY.rst:.. _02a08a0: https://github.com/galaxyproject/planemo/commit/02a08a0

$ grep -i "f3c6917" *.rst
HISTORY.rst:  `Pull Request 472`_, f3c6917_
HISTORY.rst:.. _f3c6917: https://github.com/galaxyproject/planemo/commit/f3c6917
HISTORY.rst:.. _f3c6917: https://github.com/galaxyproject/planemo/commit/f3c6917

$ grep -i "9427b47" *.rst
HISTORY.rst:  9427b47_
HISTORY.rst:.. _9427b47: https://github.com/galaxyproject/planemo/commit/9427b47
HISTORY.rst:.. _9427b47: https://github.com/galaxyproject/planemo/commit/9427b47

$ grep -i "pull request 216" *.rst
HISTORY.rst:  metadata. `Pull Request 216`_
HISTORY.rst:.. _Pull Request 216: https://github.com/galaxyproject/planemo/pull/216
HISTORY.rst:.. _Pull Request 216: https://github.com/galaxyproject/planemo/pull/216

$ grep -i "pull request 207" *.rst
HISTORY.rst:  (thanks to Peter Cock). `Pull Request 207`_, `Issue 205`_
HISTORY.rst:.. _Pull Request 207: https://github.com/galaxyproject/planemo/pull/207
HISTORY.rst:.. _Pull Request 207: https://github.com/galaxyproject/planemo/pull/207
```

Others are actually used more than once, e.g.

```
$ grep -i "8117e03" *.rst
HISTORY.rst:  files. 8117e03_
HISTORY.rst:  8117e03_
HISTORY.rst:.. _8117e03: https://github.com/galaxyproject/planemo/commit/8117e03
HISTORY.rst:.. _8117e03: https://github.com/galaxyproject/planemo/commit/8117e03

$ grep -i "pull request 99" *.rst
HISTORY.rst:  for ``shed_upload`` (thanks to Björn Grüning). `Pull Request 99`_
HISTORY.rst:* Allow ignoring file patterns in ``.shed.yml`` (thanks to Björn Grüning). `Pull Request 99`_
HISTORY.rst:.. _Pull Request 99: https://github.com/galaxyproject/planemo/pull/99
HISTORY.rst:.. _Pull Request 99: https://github.com/galaxyproject/planemo/pull/99
```

**Comment by peterjc** [comment]
I found https://pypi.python.org/pypi/restructuredtext_lint very useful, its command line tool ``rst-lint`` gives line numbers. However, it does something a little different to ``make lint-readme`` as it only seems to parse standalone RST files (not the RST strings as built up in ``setup.py``).

**Comment by peterjc** [comment]
TravisCI confirms this fixes the ``py27-lint-readme`` target, ready for merging?

**Comment by peterjc** [comment]
Fixed by merging #525.
